test/commands/generate/models.test.ts

```diff
@@ -2,14 +2,22 @@
 /* eslint-disable sonarjs/no-identical-functions */
 import path from 'path';
 import { test } from '@oclif/test';
+import { createMockServer, stopMockServer } from '../../testHelper';
 const generalOptions = ['generate:models'];
 const outputDir = './test/commands/generate/models';
 
 describe('models', () => {
+
+  beforeAll(()=>{
+    createMockServer();
```

Comment:
I was thinking that should i add some variable here for the port we are using so we can use it in the URLs .

Reply:
not sure what you mean, can you explain?